client/src/pages/NewGamePage.tsx

```diff
@@ -70,65 +70,66 @@ export const NewGamePage = observer(() => {
-                        <div className={styles.gameCreateInput}>
-                            <CreateGameForm
-                                valueRoomLimit={roomLimit}
-                                onChangeRoomLimit={onChangeRoomLimitHandler}
-                                valueGameDuration={gameDuration}
-                                onChangeGameDuration={onChangeGameDurationHandler}
-                                disabledBtn={!(nameUser && roomLimit && gameDuration)}
-                                clickHandler={startGame}
-                                titleBtn={START_GAME_LABEL}
-                            />
-                        </div>
-                    </div>
-                    <div className={styles.inputs}>
-                        <h3 className={styles.titleForInput}>List of active games</h3>
-                        <button onClick={openList} className={styles.buttonEnterName}>
-                            {START_GAME_LABEL}
-                        </button>
+                        <CreateGameForm
+                            valueRoomLimit={roomLimit}
+                            onChangeRoomLimit={onChangeRoomLimitHandler}
+                            valueGameDuration={gameDuration}
+                            onChangeGameDuration={onChangeGameDurationHandler}
+                            disabledBtn={!(nameUser && roomLimit && gameDuration)}
+                            clickHandler={startGame}
+                            titleBtn={START_GAME_LABEL}
+                        />
                     </div>
                     <div className={styles.inputs}>
                         <h3 className={styles.titleForInput}>JOIN THE GAME</h3>
-                        <div className={styles.nameInput}>
-                            <AddForm
-                                onChangeText={setJoinUrl}
-                                placeholder={'ADD HTTP'}
-                                value={joinUrl}
-                                type={'text'}
-                                disabledBtn={nameUser === ''}
-                                clickHandler={joinToGame}
-                                titleBtn={START_GAME_LABEL}
-                            />
-                        </div>
-                    </div>
-                </div>
-            ) : (
-                <>
-                    <div className={styles.nameInput}>
                         <AddForm
-                            onChangeText={setNameUser}
-                            placeholder={'Enter The name'}
-                            value={nameUser}
+                            onChangeText={setJoinUrl}
+                            placeholder={'ADD HTTP'}
+                            value={joinUrl}
                             type={'text'}
-                            disabledBtn={!nameUser}
-                            clickHandler={saveName}
+                            disabledBtn={nameUser === '' || joinUrl === ''}
```

Comment:
What do you think? Or do we necessarily need to check an empty string?
```suggestion
                            disabledBtn={!(nameUser && joinUr)}
```